script/database-roles/rds-writeonly-role.sql

```diff
@@ -0,0 +1,42 @@
+-- creates a role to allow syncing data from fivetran to a database
+DO
+$$
+  DECLARE
+    table_name     varchar;
```

Comment:
```suggestion
```
nit: in a `foreach .. in ..` loop, the loop variable is implicitly declared, so this should not be necessary